tasks/models.py

```diff
@@ -5,11 +5,29 @@
 class Task(models.Model):
     org = models.ForeignKey(Org, on_delete=models.CASCADE)
     author = models.ForeignKey(User, on_delete=models.CASCADE)
-    social_media_platform = models.CharField(max_length=30)
+    social_media_platform_choices = (
+        ('facebook', 'Facebook'),
+        ('whatsapp', 'Whatsapp'),
+        ('linkedin', 'LinkedIn'),
+        ('youtube', 'Youtube'),
+        ('instagram', 'Instagram'),
+        ('twitter', 'Twitter'),
+        ('other', 'Other')
+    )
+    social_media_platform = models.CharField(
+        max_length=30,
+        choices=social_media_platform_choices
+        )
     description = models.CharField(max_length=300)
-    share_link = models.CharField( max_length=30, null=True)
+    type_choices = (
+        ('LINK', 'LINK'),
+        ('TEXT', 'TEXT'),
+        ('IMG', 'IMG'),
+    )
+    share_type = models.CharField(max_length=5, choices=type_choices)
+    share_link = models.CharField(max_length=30, null=True)
```

Comment:
I think the link `max_length` should be increased because in some cases the links are quite big in length.

Reply:
Increased it to 100 🙌